### PR TITLE
Add Unity SDK Docs based on the docs at http://docs.swrve.com

### PR DESCRIPTION
**Pull request by johnkearney**

### CODE REVIEW

**Comment by Sergio-Mira** [comment]
:rocket:

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /packages
-/docs/
```
Comment (aoifec) [inline .gitignore]: Hi pal

**Comment by oisin** [comment]
Hi all, this is a Product call, and Shelley is the docs owner. Closing this for now, it can be re-opened later.

**Comment by johnkearney** [comment]
Thanks @oisin.

@Sergio-Mira - I've merged in release-3_7. But it looks like I've got one of your commits (https://github.com/Swrve/swrve-unity-sdk/commit/cd4f0f2fdaa7fa6cc32bd4ec697c4f46457170b2) from 3_5 in here too - changes to AndroidManifest.xml. Should that have been merged forward from 3_5.

**Comment by Sergio-Mira** [comment]
Thats fine, it was an improvement to what we had on the AndroidManifest.xml before

**Pull request closed without merging**